gunshot/data-processing-application/CSV.java

```diff
@@ -0,0 +1,108 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+/**
+ * CSV Class
+ * 
+ * <p>CSV is a class for the reading of a dataset in a csv format
+ * 
+ * @author Carver Forbes
+ * @version 0.1
+ */
+public class CSV {
+  private File file;
+
+  /**
+   * CSV constructor
+   * 
+   * @param pathname the pathname for the csv file to be read
+   * @throws IllegalArgumentException if the pathname is incorrect
+   */
+  public CSV(String pathname) throws IllegalArgumentException {
+    file = new File(pathname);
+    //TODO: handle permission issues
```

Comment:
We can remove this since we're mainly using this locally. We can address this when an issue actually comes up.